### DISCUSSION TITLE
弹出选择 Popselect

### DISCUSSION THREAD

**Discussion by 994995** [tusen-ai/naive-ui#2650] [Q&A]
![image](https://user-images.githubusercontent.com/70012946/158923913-7afe4d99-f1d6-41b4-8576-50356c8a2121.png)
点击更多想要关闭了Popselect改如何操作呢

**Comment by whale2002** [comment]
可以给 `n-popselect` 绑定一个 `show` 参数来控制，点击 `查看更多以后` show 置为 false

**Reply by 07akioni** [reply]
这是一个方法。

我看了一下源码，现在 popconfirm 和 popselect 都没有暴露出 popover 的 setShow 和 syncPosition 方法，下个版本会加上。

加上之后可以通过 ref 调用 setShow 方法。
